apm-sniffer/apm-sdk-plugin/hbase-1.x-2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/hbase/define/HTableInstrumentation.java

```diff
@@ -47,11 +47,15 @@
 import static net.bytebuddy.matcher.ElementMatchers.named;
 import static net.bytebuddy.matcher.ElementMatchers.takesArgument;
 import static net.bytebuddy.matcher.ElementMatchers.takesArguments;
+import static org.apache.skywalking.apm.agent.core.plugin.bytebuddy.ArgumentTypeNameMatch.takesArgumentWithType;
 
 public class HTableInstrumentation extends ClassInstanceMethodsEnhancePluginDefine {
```

Comment:
Add the comments here about https://github.com/apache/skywalking/pull/6577#discussion_r596931939. We need to be clear about the naming rules.